internal/ifelse/branch.go

```diff
@@ -102,6 +102,8 @@ func (b Branch) IsShort() bool {
 		return true
 	case 1:
 		return isShortStmt(b.block[0])
+	case 2:
+		return isShortStmt(b.block[1])
```

Comment:
If I understand well, if `b.block` is

```go
[
  { // this is b.block[0]
    println()
    println()
    println()
    println()
  }, 
  println() // this is b.block[1]
]
```

`IsShort` will return `true` ?

Reply:
Do you mean the following?

```go
func fn17() {
	if err := a(); err != nil {
		{ // this is b.block[0]
			println()
			println()
			println()
			println()
		}
		println() // this is b.block[1]
	}
}
```


If so, early-return doesn't handle this case. It's quite uncommon to see code like this in the wild.

Reply:
> Is that syntax valid? 🤯 I have never faced it.

Yes, it's valid.